app/components/CircularProgressBar.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import React from "react";
+import { useTheme } from "styled-components";
+
+const cleanPercentage = (percentage) => {
+  const tooLow = !Number.isFinite(+percentage) || percentage < 0;
+  const tooHigh = percentage > 100;
+  return tooLow ? 0 : tooHigh ? 100 : +percentage;
+};
+
+const Circle = ({ colour, pct }: { colour: string, pct?: number }) => {
```

Comment:
- US spelling for `color` pls, I'm English so it kills me a little on the inside, but it's better to be consistent. 
- `pct` => `percentage`